src/Joomla/Form/Form.php

```diff
@@ -766,8 +766,8 @@ public function loadFile($file, $reset = true, $xpath = false)
 			}
 		}
 
-		// Attempt to load the XML file.
-		$xml = simplexml_load_file($file);
+		// Attempt to load the XML files.
+		$xml = simplexml_load_file($file, 'SimpleXMLElement', LIBXML_NOENT);
```

Comment:
I understand that this is no place to discuss things, but I could not think of any appropriate place for clarification of my doubt.
@photodude As far as I understand `LIBXML_NOENT` makes simple xml to substitute entities in the XML. Can you explain how that is significant with "Adding XML include file functionality"?